ui/src/dashboards/reducers/ui.js

```diff
@@ -109,6 +109,27 @@ export default function ui(state = initialState, action) {
       return {...state, ...newState}
     }
 
+    case 'DELETE_DASHBOARD_CELL': {
+      const {cell} = action.payload
+      const {dashboard} = state
+
+      const index = dashboard.cells.findIndex((c) => c.x === cell.x && c.y === cell.y)
```

Comment:
But given that you're already iterating to find an index, you may as well simply filter. Otherwise you're iterating an extra index length over the array.